fission-web-server/library/Fission/Web/Server/Auth/Token/Basic.hs

```diff
@@ -88,6 +88,6 @@ checkUser (BasicAuthData rawUsername password) =
 
 parseBasic :: Auth.Basic.Token -> Either Auth.Error BasicAuthData
 parseBasic (Auth.Basic.Token token) =
-  case Ch.split ':' (Base64.decodeBase64Lenient token) of
+  case Ch.split ':' (Base64.decodeLenient token) of
```

Comment:
There's a few of these around. It's from updating a dependency. Reads better, so okay by me 👍